### DISCUSSION TITLE
Scope of this project

### DISCUSSION THREAD

**Discussion by Mitgenosse** [reactivemarkets/react-financial-charts#490] [Ideas]
Hello there,

for the past days I've been looking into this project as an alternative to tradingview. Tradingview obviously has a lot of features, so I'm wondering about where react-financial-charts is heading.

Up to this point I see that `react-financial-charts` has a lot of flexibility and customization options, which is nice. But it seems like you still need to do a lot to get some kind of nicely working charts.

Is there a chance that we get to the point where there is some kind of high order component that you just need to feed the chart data into, and that handles chart-related features by itself. By chart-related I mean things like drawing trend lines and horizontal lines/having a toolbar with those functions, being able to switch between candles/normal lines etc., adding indicators like EMA, etc.

The only thing a dev would need is to handle the data feed from any1 API.

What do you think?

**Comment by markmcdowell** [comment]
We definitely want to simplify usage, part of the reason this project is setup as a monorepo is so we can add wrappers without bloating the core components.

so yes I think that should be on the roadmap to add toolbars etc similar to the utils package
